examples/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"fmt"
+	"github.com/surrealdb/surrealdb.go"
+)
+
+// TODO: add in ability to do INFO command on database
+// TODO: should let users specify a selector other than '*' for select statements
+// TODO: set up docker container so it also allows the client to sign in
+func main() {
+
+	println("Step 1: Connect to SurrealDB")
```

Comment:
Use `fmt.Println` since `println` is not guaranteed to stay in the language.

Reply:
Updated